beacon-chain/state/state-native/v1/state_trie.go

```diff
@@ -72,46 +112,89 @@ func InitializeFromProtoUnsafe(st *ethpb.BeaconState) (*BeaconState, error) {
 	return b, nil
 }
 
-// Copy returns a deep copy of the beacon state.
-func (b *BeaconState) Copy() state.BeaconState {
-	if !b.hasInnerState() {
-		return nil
+func Initialize() (*BeaconState, error) {
```

Comment:
Godoc for this, should be a function comment on what it does.

Reply:
Also why is this special method needed

Reply:
It is most likely not needed anymore.